config/application.yml.default

```diff
@@ -138,6 +138,7 @@ in_person_enrollments_ready_job_queue_url: ''
 in_person_enrollments_ready_job_visibility_timeout_seconds: 30
 in_person_enrollments_ready_job_wait_time_seconds: 20
 in_person_full_address_entry_enabled: true
+in_person_opt_in_available_completion_survey_url: 'https://handbook.login.gov'
```

Comment:
I still need to add this new config variable to all deployed environments.  I will screenshare with other Joy engineers while making the change.  We can't merge this PR until that work is done.

Reply:
I added this new config variable to all deployed environments yesterday.  I screenshared with other Joy engineers during the call. ✅